src/_guides/openapi/specification/v3.1/the-perfect-modern-openapi-workflow.md

```diff
@@ -0,0 +1,660 @@
+
+### API Governance
+
+API Governance is the framework or making great, consistent, reliable, and
+consumable APIs. Over the last decade, API governance has gone from some vague
+concept being mentioned at conferences, to being a beautifully solved problem
+with a plethora of tooling. 
+
+Whilst [API governance is a massive
+topic](_guides/openapi/leveraging-the-openapi-specification-for-api-governance.md),
+a few key parts can be handled with OpenAPI tooling, speeding up and partially
+automating "API Design Reviews", simplifying change detection, automation of
+"API style guides", and creating an API Catalog that can help keep track of all
+the APIs in a company so they don't go missing and fall into disrepair. 
+
+#### API Design Reviews
+
+Just as pull requests need to pass automated testing as well as being peer
+reviewed, OpenAPI changes need to be reviewed to make sure they are a good idea
+for the API, consumers, and organization at large. Design reviews make sure the
+API is going to be useful not just valid. By making sure the OpenAPI is
+suggesting a good change, more mistakes will be caught before they make it into
+production. 
+
+This can be a very manual and time consuming process without OpenAPI as there is
+a lot of code review, and the folks helping with design reviews might have to
+learn various languages, multiple frameworks, and understand all the intricacies
+of those frameworks less obvious conventions. Moving the review to OpenAPI
+removes all of that, and helps to democratize and centralize the process and
+have more folks involved, instead of just those who understand the code.
+
+To design reviews easier, Bump.sh can show a list of relevant changes between
+the OpenAPI in the a pull request and the latest deployed document. This helps
+skip staring at infinite lines of YAML/JSON, trying to understand what, if
+anything, meaningfully changed. New properties, changed property validations, or
+breaking changes like a removed endpoint or new required property. 
+
+```yaml
+# .github/workflows/api-diff.yaml
+name: API Changes
+permissions:
+  contents: read
+  pull-requests: write
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  changes:
+    name: Detect and preview API changes
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Comment pull request with API diff
+        uses: bump-sh/github-action@v1
+        with:
+          doc: <BUMP_DOC_ID>
+          token: ${{secrets.BUMP_TOKEN}}
+          file: api/openapi.yaml
+          command: diff
+
+      - name: API Preview pull request with API diff
+        uses: bump-sh/github-action@v1
+        with:
+          doc: <BUMP_DOC_ID>
+          token: ${{secrets.BUMP_TOKEN}}
+          file: api/openapi.yaml
+          command: preview
```

Comment:
We already [add a `preview documentation` link in the diff comment](https://github.com/bump-sh/github-action/blob/master/src/diff.ts#L41-L49). So this shouldn't be necessary.

Please confirm if you don't see the preview links?